app/models/order/order.server.tsx

```diff
@@ -16,13 +16,15 @@ export const findOrderById = async (id: string): Promise<Order | null> => {
 
 export const createOrder = async (data: OrderInput): Promise<Order> => {
   return database.order.create({
-    data,
+    data: {
+      ...data,
+    },
```

Comment:
Why this changed?

Reply:
oh, my bad, something i changed when trying to resolve an issue, didn't mean to push this